test/fixtures/assignments.yml

```diff
@@ -2,12 +2,10 @@
 
```

Comment:
✂️ use factory bot

Reply:
@tachyons , these were changes from the integration tests which are longer being used.So i have removed these changes from this PR.